### ISSUE TITLE
Review records with missing ApplicationID in ODS

### ISSUE THREAD

**Issue by T854236** [bcgov/digital-journeys#867]
We see records with missing ApplicationID in ODS.  Those records does have GUI/IDIR information.

**Comment by T854236** [comment]
Loaded file of records in SharePoint: https://bcgov.sharepoint.com/:x:/r/teams/03991/Shared%20Documents/Data%20Collection/Telework_submissions_with_missing_applicationID_Jan20_2023.csv?d=w63395c28273e4080a820b8b07cb00d67&csf=1&web=1&e=Q2W7Ie

**Comment by iman-jamali-fw** [comment]
@T854236 

It's been very difficult to find these submissions in the DGJ. Are there any other pieces of data (even the date of submission) that I can use to reference?

So far, I was only able to find a few in-progress submissions from those users who are not related to the list you shared :(

**Comment by T854236** [comment]
@iman-jamali-fw 

A different issue, can you take a look at this spreadsheet?  We found some records with no schedule but the NoFormalSchedule is false.  I believe when that is false, it should have a schedule?

https://bcgov.sharepoint.com/:x:/r/teams/03991/Shared%20Documents/Data%20Collection/[Telework_submissions_with_missing_schedule_Jan24_2023.csv](https://bcgov.sharepoint.com/:x:/r/teams/03991/Shared%20Documents/Data%20Collection/Telework_submissions_with_missing_schedule_Jan24_2023.csv?d=w51924d15ca13404b85cb54832b82171c&csf=1&web=1&e=IMKgXS)?d=w51924d15ca13404b85cb54832b82171c&csf=1&web=1&e=IMKgXS

**Comment by iman-jamali-fw** [comment]
@T854236 

You're right. I tested some of these submission and they all had schedule data.

**Comment by T854236** [comment]
@iman-jamali-fw 

IDIR and GUID information is there.  Just the schedule is missing.

**Comment by iman-jamali-fw** [comment]
@T854236 
Hmm, interesting! I'll do an investigation... in the meantime would you like me to find the missing schedule?

**Comment by T854236** [comment]
@iman-jamali-fw 

Yes please that would be great!

**Comment by iman-jamali-fw** [comment]
@T854236 

Sent you an email with the missing schedule values.

**Comment by MeghanStothers** [comment]
Patrick will be pushing this to prod tomorrow.

**Comment by Abuchana** [comment]
Patrick confirmed deployed to Prod
